docs/basics/101-130-yodaproject.rst

```diff
@@ -0,0 +1,564 @@
+A YODA-compliant data analysis project with DataLad
+---------------------------------------------------
+
+Now that you know about the YODA principles and DataLad's Python API, it is
+time to start working on ``DataLad-101``'s midterm project.
+In order to combine the previous sections, this midterm project is a
+YODA-compliant data analysis project, written in Python, and set up in a
+DataLad dataset.
+
+For your submission, you decide to analyse the
+`iris flower data set <https://en.wikipedia.org/wiki/Iris_flower_data_set>`_.
+It is a multivariate data set on 50 samples of each of three species of Iris
+flowers (*Setosa*, *Versicolor*, or *Virginica*), with four variables: the length and width of the sepals and petals
+of the flowers in centimeters. It is often used in introductory data science
+courses for statistical classification techniques in machine learning, and
+widely available, among many other sources as
+`Github gists <https://gist.github.com/netj/8836201>`_.
```

Comment:
This is a good source that also comes with code and a plot: https://scikit-learn.org/stable/auto_examples/datasets/plot_iris_dataset.html

Reply:
Ah, I understand now that the file is the important part...